test/user.test.js

```diff
@@ -429,7 +426,6 @@ describe('User', function() {
         User.login({email: 'b@c.com', password: pass72Char}, function(err, accessToken) {
           if (err) return done(err);
           assertGoodToken(accessToken);
-          assert(accessToken.id);
```

Comment:
`assert(accessToken.id)` is in fact already comprised in
`assertGoodToken(accessToken)`:

```
function assertGoodToken(accessToken) {
    assert(accessToken.userId);
    assert(accessToken.id);
    assert.equal(accessToken.id.length, 64);
  }
```
so the second assert is redundant

similar logic below where i simplified numerous assert with the existing assertGoodToken helper